tests/e2e/lib.sh

```diff
@@ -152,6 +152,7 @@ export_test_environment() {
     ci_export ROX_DECLARATIVE_CONFIGURATION "${ROX_DECLARATIVE_CONFIGURATION:-true}"
     ci_export ROX_COMPLIANCE_ENHANCEMENTS "${ROX_COMPLIANCE_ENHANCEMENTS:-true}"
     ci_export ROX_POLICY_CRITERIA_MODAL "${ROX_POLICY_CRITERIA_MODAL:-true}"
+    ci_export ROX_POLICY_VIOLATIONS_ADVANCED_FILTERS "${ROX_POLICY_VIOLATIONS_ADVANCED_FILTERS:-true}"
```

Comment:
Saif, can. you make corresponding change in a new place for operator environment around line 295

See second bullet point of step 4
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/README.md#add-a-feature-flag-to-frontend-code